courses/free-courses-fa_IR.md

```diff
@@ -34,6 +36,13 @@
 * [درس اصول برنامه‌نویسی سی و سی پلاس پلاس از دانشگاه صنعتی اصفهان](https://maktabkhooneh.org/course/%D8%A7%D8%B5%D9%88%D9%84-%D8%A8%D8%B1%D9%86%D8%A7%D9%85%D9%87-%D9%86%D9%88%DB%8C%D8%B3%DB%8C-C-%D9%88-C-mk68) - Kiarash Bazargan
 
 
+### C-Sharp
+
+#### DotNetCore
+* [دوره مقدماتی دات نت 5 تحت وب](https://bugeto.net/courses/free-introductory-asp-dot-net-core-training-course) - Ehsan Babaei *(نیاز به ثبت نام دارد)*
+
+
+
```

Comment:
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.